pom.xml

```diff
@@ -118,7 +118,7 @@
         <junit5.version>5.4.2</junit5.version>
         <hamcrest.version>1.3</hamcrest.version>
         <mockito.version>1.10.19</mockito.version>
-        <mockito2.version>2.28.2</mockito2.version>
+        <mockito3.version>3.7.0</mockito2.version>
```

Comment:
Looks like there's an error in the closing tag; it needs to be `</mockito3.version>`